packages/components/aside/stories/aside.stories.js

```diff
@@ -0,0 +1,8 @@
+import { storiesOf } from '@storybook/html';
+import '@tradeshift/elements';
+import '@tradeshift/elements.aside';
+
+storiesOf('ts-aside', module).add('type=""', () => {
+	const aside = document.createElement('ts-aside');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "aside". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1488)